projectforge-business/src/main/resources/flyway/migrate/common/V7.4.1.0__7.4.1.0-RELEASE-VACATION_OTHER_REPLACEMENTS.sql

```diff
@@ -1,9 +1,9 @@
--- Vacation forms of employees supports now others substitutes (replacements) in addition to the main substitue (replacement)
+-- Vacation forms of employees now supports others substitutes (replacements) in addition to the main substitute (replacement)
 
 CREATE TABLE t_employee_vacation_other_replacements
 (
     vacation_id INTEGER NOT NULL,
-    employee_id     INTEGER NOT NULL
+    employee_id INTEGER NOT NULL
 );
```

Comment:
Please examine source change

Reply:
Can't change this. It's a flyway migration script and can't be changed afterwards (due to hashsum).

Reply:
Apologies. I've removed the file from the commit.